go-console/.vscode/tasks.json

```diff
@@ -0,0 +1,919 @@
+      }
+    },
+    {
+      "label": "wait-a-bit",
+      "detail": "hide",
+      "command": "sleep",
+      "type": "process",
+      "args": ["1"],
+      "dependsOrder": "sequence",
+      "problemMatcher": "$msCompile",
+      "icon": {
+        "id": "layers",
+        "color": "terminal.ansiCyan"
+      }
+    },
+    {
+      "label": "pos-cleanup",
+      "detail": "hide",
+      "command": "ssh",
+      "type": "process",
+      "args": [
+        "-i",
+        "${workspaceFolder}/.conf/id_rsa",
+        "-p",
+        "${config:torizon_debug_ssh_port}",
+        "-o",
+        "UserKnownHostsFile=/dev/null",
+        "-o",
+        "StrictHostKeyChecking=no",
+        "torizon@${config:torizon_ip}",
+        "rm -rf ~/app"
+      ],
+      "dependsOrder": "sequence",
+      "problemMatcher": "$msCompile",
+      "icon": {
+        "id": "layers",
+        "color": "terminal.ansiCyan"
+      }
+    },
+    {
+      "label": "validate-settings",
+      "detail": "hide",
+      "command": "bash",
+      "type": "process",
+      "args": ["-c", "[[ ! -z \"${config:torizon_ip}\" ]] && true || false"],
+      "dependsOrder": "sequence",
+      "problemMatcher": "$msCompile",
+      "icon": {
+        "id": "layers",
+        "color": "terminal.ansiCyan"
+      }
+    },
+    {
+      "label": "validate-arch-x86",
+      "detail": "hide",
+      "command": "bash",
+      "type": "process",
+      "args": [
+        "-c",
+        "[[ \"${config:torizon_arch}\" == \"x86_64\" ]] && true || false"
+      ],
+      "dependsOrder": "sequence",
+      "problemMatcher": "$msCompile",
+      "icon": {
+        "id": "layers",
+        "color": "terminal.ansiCyan"
+      }
+    },
+    {
+      "label": "validate-arch-arm",
+      "detail": "hide",
+      "command": "bash",
+      "type": "process",
+      "args": [
+        "-c",
+        "[[ \"${config:torizon_arch}\" == \"armhf\" ]] && true || false"
+      ],
+      "dependsOrder": "sequence",
+      "problemMatcher": "$msCompile",
+      "icon": {
+        "id": "layers",
+        "color": "terminal.ansiCyan"
+      }
+    },
+    {
+      "label": "validate-arch-arm64",
+      "detail": "hide",
+      "command": "bash",
+      "type": "process",
+      "args": [
+        "-c",
+        "[[ \"${config:torizon_arch}\" == \"aarch64\" ]] && true || false"
+      ],
+      "dependsOrder": "sequence",
+      "problemMatcher": "$msCompile",
+      "icon": {
+        "id": "layers",
+        "color": "terminal.ansiCyan"
+      }
+    },
+    {
+      "label": "validate-arch-riscv64",
+      "detail": "hide",
+      "command": "bash",
+      "type": "process",
+      "args": [
+        "-c",
+        "[[ \"${config:torizon_arch}\" == \"riscv64\" ]] && true || false"
+      ],
+      "dependsOrder": "sequence",
+      "problemMatcher": "$msCompile",
+      "icon": {
+        "id": "layers",
+        "color": "terminal.ansiCyan"
+      }
+    },
+    {
+      "label": "wait-debugpy-start-arm64",
```

Comment:
Where it is written `"wait-debugpy-start-arm64"` here on the `tasks.json` and also on `launch.json` it can be changed to something like `"wait-debugger-start-arm64"`